mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -12,22 +12,27 @@ mailing emails.
 Configuration
 =============
 
-To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
-and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
-of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+To configure unsubscribe label go to *Setting > Technical > Parameters > System
+parameters* and add a 'mass_mailing.unsubscribe.label' parameter with html to
```

Comment:
This is nothing to deal with translation. noupdate=1 makes the parameter overwritable and you don't lose the custom value when updating.